libraries/tools/kotlin-gradle-plugin/src/common/kotlin/org/jetbrains/kotlin/gradle/targets/js/testing/karma/KotlinKarma.kt

```diff
@@ -93,11 +95,41 @@ class KotlinKarma(
         useMocha()
         useWebpack()
         useSourceMapSupport()
+        usePropBrowsers()
 
         // necessary for debug as a fallback when no debuggable browsers found
         addChromeLauncher()
     }
 
+    private fun usePropBrowsers() {
+        val propKey = "kotlin.js.browser.karma.browsers"
+        val propValue = project.findCompositeProperty("$propKey.${compilation.target.name}") ?: project.findCompositeProperty(propKey)
+        val propBrowsers = propValue?.toString()?.split(",")
+        propBrowsers?.forEach {
+            when (it.trim().toLowerCase()) {
+                "chrome" -> useChrome()
+                "chrome-canary" -> useChromeCanary()
+                "chrome-canary-headless" -> useChromeCanaryHeadless()
+                "chrome-headless" -> useChromeHeadless()
+                "chrome-headless-no-sandbox" -> useChromeHeadlessNoSandbox()
+                "chromium" -> useChromium()
+                "chromium-headless" -> useChromiumHeadless()
+                "firefox" -> useFirefox()
+                "firefox-aurora" -> useFirefoxAurora()
+                "firefox-aurora-headless" -> useFirefoxAuroraHeadless()
+                "firefox-developer" -> useFirefoxDeveloper()
+                "firefox-developer-headless" -> useFirefoxDeveloperHeadless()
+                "firefox-headless" -> useFirefoxHeadless()
+                "firefox-nightly" -> useFirefoxNightly()
+                "firefox-nightly-headless" -> useFirefoxNightlyHeadless()
+                "ie" -> useIe()
+                "opera" -> useOpera()
+                "phantom-js" -> usePhantomJS()
+                "safari" -> useSafari()
```

Comment:
May it will be good to add else clause with an error or warning here, so misspelled words will be shown to user (with suggestion of available values), as not everyone would know all names

Reply:
Good tip, updated